android/sdl_android/src/androidTest/java/com/smartdevicelink/test/TestValues.java

```diff
@@ -1133,6 +1137,9 @@ public void onVoiceCommandSelected() {
 		GENERAL_WINDOW_STATUS.setLocation(TestValues.GENERAL_GRID);
```

Comment:
@kostyaBoss On line 908, I think you need to add these lines:

`GENERAL_VIDEOSTREAMINGCAPABILITY.setAdditionalVideoStreamingCapabilities(...);`
`GENERAL_VIDEOSTREAMINGCAPABILITY.setDiagonalScreenSize(...);`
`GENERAL_VIDEOSTREAMINGCAPABILITY.setPixelPerInch(...);`
`GENERAL_VIDEOSTREAMINGCAPABILITY.setScale(...);`